README.md

```diff
@@ -9,3 +10,7 @@ might help with the notation for mathematical expressions.
 $G_1=(V_1 , E_1)$ is isomorphic to $G_2 = (V_2, E_2)$ if there exists a
 one-to-one and onto function (bijection) $f: V_1 \rightarrow V_2$ such that $(u,v)
 \in E_1$ iff $(f(u),f(v)) \in E_2$.
+
+## Proof:
+
+In order for a graph to be completely connected each vertex must have an edge that leads to every other vertex which means that ever vertex in the graph must have the same degree. Since we know that $A$ and $B$ have the same number of nodes and we know that each vertex within both of those graphs has the same number of degrees then any vertex in $A$ can be mapped to any vertex in $B$ and from there these graphs can be fully mapped to eachother and are isomorphic.
```

Comment:
Not clear -- what is this mapping? And the definition we've used doesn't talk about node degrees.